src/main/java/org/deepsymmetry/beatlink/data/MetadataFinder.java

```diff
@@ -541,7 +541,7 @@ private void recordMount(SlotReference slot) {
         if (mediaMounts.add(slot)) {
             deliverMountUpdate(slot, true);
         }
-        if (!mediaDetails.containsKey(slot)) {
+        if (!mediaDetails.containsKey(slot) && !VirtualRekordbox.getInstance().isRunning()) {
```

Comment:
It’s probably a bit more informative to use `!VirtualCdj.getInstance().inOpusQuadCompatibilityMode()`?

Reply:
Sure, done!